_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,118 @@
+
+- **BigPipe render pipeline**. Sends pages in a way that allows browsers to show them much faster. First sends the cacheable parts of the page, then the dynamic/uncacheable parts. Uses the [BigPipe technique](https://www.facebook.com/notes/facebook-engineering/bigpipe-pipelining-web-pages-for-high-performance/389414033919).
+
+These improvements have the potential to make your Drupal 8 website fly! And if after all that it is not “flying” - than you need someone to review the code that powers your website’s features. ([Contact us](/contact/))
+
+#### Semantic Markup
+
+Search engines appreciate clean markup that explicitly describes the purpose of on-page elements. Thanks to [HTML5 Initiative](https://www.drupal.org/community-initiatives/drupal-core/html5) for Drupal 8 development, we now have a number of great markup improvements right in Drupal core:
+
+- HTML5 themes with new semantic elements in core templates
+
+- Support for the new form elements to Drupal's Form API
+
+- Rich media handling with \<video> and \<audio> elements
+
+- ARIA roles in markup to improve accessibility
+
+- Twig theming engine - makes it harder for developers to create messy, non-semantic code
+
+#### Content as a Service
+
+Another exciting new feature of Drupal 8 is a flexible content delivery.
+
+It enables Drupal to create and deliver content as a service to any channel (aggregators, apps. other sites, etc.) or device. No longer is your content restricted to residing on your website only. Decoupled, or “headless” Drupal can power your apps while letting you control all the content from a unified admin interface.
+
+#### Drupal 8  Multilingual Capabilities
+
+Previous versions of Drupal had partial support for multilingual websites. Luckily, Drupal 8 had a fundamental overhaul of its [multilingual system](http://www.drupal8multilingual.org/features). Every single component is translatable out of the box in Drupal core without any additional modules. Drupal core natively supports 94 languages. Also, the administration interface is now entirely translatable. Media assets (files or images), can now be assigned to a language or shared between languages. This gives a huge advantage to businesses that aim to reach global audience.
+
+SEO for Drupal 8 is off to a good start with just the core features! Drupal 8 also has a growing number of contributed modules that can amplify your SEO efforts. Just to name a few: [Metatags](https://www.drupal.org/project/metatag), [Google Analytics](https://www.drupal.org/project/google_analytics), [Pathauto](https://www.drupal.org/project/pathauto), [Redirect](https://www.drupal.org/project/redirect), and more.
+
+*****
+
+**Drupal 8 satisfies current SEO trends enabling marketers do their job effectively and efficiently! Even with minimal SEO, Drupal 8 ensures future marketing performance.**
```

Comment:
I'm a little confused by this bolded bottom paragraph. Is it meant to be a quick summary and call to action? I like that idea. In any case, I'm not sure what you mean by "Even with minimal SEO..." Maybe even with minimal client configuration, or alteration to what comes out of the box?